lib/puppet/util/posix.rb

```diff
@@ -1,6 +1,39 @@
 # Utility methods for interacting with POSIX objects; mostly user and group
 module Puppet::Util::POSIX
 
+  module GrpLibC
```

Comment:
Dumb question maybe, but don't we need to require `puppet/ffi/posix` in this file?

Reply:
It's required in the `begin/rescue` block of the `groups_of` method for platforms without FFI (`ffi` is required in said file) to fallback automatically to previous implementation.